types/testingutils/ssv_msgs_versioned.go

```diff
@@ -0,0 +1,263 @@
+package testingutils
+
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/herumi/bls-eth-go-binary/bls"
+
+	"github.com/bloxapp/ssv-spec/types"
+)
+
+var TestProposerConsensusDataV = func(version spec.DataVersion) *types.ConsensusData {
```

Comment:
All the functions in this file are currently being used in tests?
Meaning we can know they are ok since roots didn't change?

Reply:
almost
there is a function `PreConsensusRandaoNoMsgV` that there is no usage for it in ssv_msgs as well